views/block.tmpl

```diff
@@ -132,7 +132,7 @@
                     <tr>
                         <td class="text-right lh1rem pr-2 xs-w117">TIME</td>
                         <td class="lh1rem" style="min-width: 178px;">
-                            ({{timezone}}) {{.BlockTime}}<span class="op60 fs12 nowrap jsonly"> (<span data-target="time.age" data-age="{{.BlockTime.RFC3339}}"></span> ago)</span>
+                            (UTC) {{.BlockTime}}<span class="op60 fs12 nowrap jsonly"> (<span data-target="time.age" data-age="{{.BlockTime.RFC3339}}"></span> ago)</span>
```

Comment:
Why switch back to hardcoding UTC here?
Seems like we can simply keep `({{timezone}}) {{.BlockTime}}` since current setup does support the backend running in any timezone.

Reply:
`{{timezone}}` is the timezone of the server, which is not relevant to the client.  We want to show a UTC time regardless of the server time zone and the client timezone.

I have yet to push another commit to the go code to make that happen correctly.  The commit I have in preparation (it's very short, I just haven't gotten back to it) is making the sql date-time reader (`sql.Scanner`) work the exact reverse to the writer (`sql.Valuer`).  The essence of the remaining issue is that the writer takes a UTC time and saves it to the DB table without a anything indicating the timezone, but the reader processes the time and assumes it is local time.

Reply:
Actually, for backward compatibility with existing table data, I'm not going to change what's written to the table until we do an actual scheme upgrade.  I'll leave the original data in the table, which has local times using the default valuer for time.Time when no timeone is permitted in the DB datatype, and also leave the default time.Time scanner, which loads the time as local time.  This at least is consistent.  The frontend solution in this case is simply to convert the time string to UTC when constructing the TimeDef for the explorer frontend.